### PR TITLE
User can see tasks marked on map as markers and as list

### PR DESCRIPTION
**Pull request by robin-lillqvist**
# [PT Story](https://www.pivotaltracker.com/story/show/172371477)
```
As Helper
In order to know who to help and where
I want to see their icon in the google maps.
```
## Changes proposed in this pull request:
* Showing 
## What I have learned working on this feature:
* Adding parameters that is required
## Screenshots (Only if client side story)
Screenshot 2020-04-19 at 14 22 55

Screenshot 2020-04-19 at 16 01 55

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ import axios from "axios";
 import { Provider } from "react-redux";
 import configureStore from "./state/store/configureStore";
 
-// axios.defaults.baseURL = "http://localhost:3000/api/v1/";
-axios.defaults.baseURL = "https://the-helping-hand.herokuapp.com/api/v1/";
+axios.defaults.baseURL = "http://localhost:3000/api/v1/";
```
Comment (Carltesio) [inline src/index.js]: Should this be the heroku Baseurl?

File: src/modules/authentication.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import JtockAuth from "j-tockauth";
 
 const auth = new JtockAuth({
-  host: "https://the-helping-hand.herokuapp.com",
-  // host: "http://localhost:3000",
+  // host: "https://the-helping-hand.herokuapp.com",
+  host: "http://localhost:3000",
```
Comment (Carltesio) [inline src/modules/authentication.js]: Should this be the heroku Baseurl?

File: cypress/integration/UserCanSeeRequestList.feature.js [hunk 3]
```diff
@@ -13,8 +13,8 @@ describe("When products are visible", () => {
     });
     cy.route({
       method: "GET",
-      url: "**/products",
-      response: "fixture:products.json",
+      url: "**/tasks",
```
Comment (JaimeCrz) [inline cypress/integration/UserCanSeeRequestList.feature.js:16]: Remember to push the test that's gonna be used first before updating it.

File: cypress/integration/UserCanSeeRequestList.feature.js [hunk 4]
```diff
@@ -13,8 +13,8 @@ describe("When products are visible", () => {
...
+      response: "fixture:taskIndex_response.json",
```
Comment (janko101) [inline cypress/integration/UserCanSeeRequestList.feature.js]: Name of the file is kinda weird

File: cypress/integration/UserCanSeeRequestList.feature.js (hunk 3 above)
Comment (robin-lillqvist) [reply cypress/integration/UserCanSeeRequestList.feature.js:16]: Done

File: src/index.js (hunk 1 above)
Comment (robin-lillqvist) [reply src/index.js]: Done

**Review by Carltesio — APPROVED** [review]

**Pull request merged**